src/components/Tour.vue

```diff
@@ -44,11 +44,45 @@ export default {
 				},
 				'ide-workspace': {
 					title: 'User Workspace',
-					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services and files.'
+					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services, custom processes and files.'
 				},
 				'ide-viewer': {
 					title: 'Data Viewer',
-					content: 'This area you can use to preview collections or inspect the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+					content: 'This area is for previewing collections or inspecting the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+				},
+
+				'connect-url': {
+					title: 'URL input',
+					content: `Paste the URL of the server you wish to connect to into this field. If you don't know any URL, you may head over to <a href="https://hub.openeo.org/">openEO Hub</a> to discover available servers.`
+				},
+				'connect-history': {
+					title: 'History of URLs',
+					content: 'Clicking on this icon will open a window that displays all URLs that you have connected to in the past (provided your browser permits storing in Local Storage).'
+				},
+
+				'login-options': {
+					target: '#credentials > .tabsHeader',
+					title: 'Login Options',
+					content: `You have several ways of authenticating with the server. The standard method is OpenID Connect (OIDC). If supported, you will see a button for each available OIDC provider here. Click on one to open its login form.`
+				},
+				'login-internal': {
+					target: '#credentials > .tabsHeader > button[title=Internal]',
+					title: 'User/Password',
+					content: `Some servers support basic authentication with a simple username-password combination. This method is discouraged and generally only used for internal development purposes (hence the name of this button). We suggest to use OpenID Connect if you can.`
+				},
+				'login-without': {
+					target: '#credentials > .tabsHeader > button[title="No credentials"]',
+					title: 'Without login',
+					content: `It is possible to proceed without any authentication, however, usually this means that you won't able to actually submit calculations etc. You can think of it as an "discovery only" mode.`
```

Comment:
```suggestion
					content: `It is possible to proceed without credentials, however, this means that you can only explore the service offerings in a read-only mode and thus you won't able to actually access the actual EO data, submit calculations or store any data.`
```

Reply:
I would avoid the term "read-only" because you can think of calculations as just a read too. After all, non of the data on the server is altered. That's why I chose "discovery only".

Reply:
Although I think read-only is okay here, I've removed the part about it completely. It did not really add a lot anyway.

Reply:
I used "without any authentication" as a different wording of "no credentials" since that is already the label of the button. I like it better from a stylistic point of view, and on top of that it has the benefit that if someone doesn't understand what the label means, a different wording might help (especially for non-English speakers who might know the somewhat more elaborate term "credentials").

I also corrected ("be"), changed ("thus"->"but") and added (`<em>`) some things for this suggestion:

```suggestion
					content: `It is possible to proceed without any authentication, however, this means that you can only <em>explore</em> the service offerings but won't be able to actually access the actual EO data, submit calculations or store any data.`
```

Reply:
Sure, please correct any mistakes I have in my suggestions :-)

Lastly, maybe we go even one step further and remove both relatively technical terms (credentials/authentication) and just say:

```suggestion
					content: `It is possible to proceed without log in, however, this means that you can only <em>explore</em> the service offerings but won't be able to actually access the actual EO data, submit calculations or store any data.`
```
Not sure which is correct though... `without log in` or `without logging in` or `without being logged in`

Reply:
:+1:
"without logging in" sounds best to me